contributors/design-proposals/auditing.md

```diff
@@ -155,7 +155,11 @@ const (
 
 The audit level is determined by the policy, which maps a combination of requesting user, namespace, verb, API group, and resource. The policy is described in detail [below](#policy).
 
-When the http request is processed, the request handler will close the audit event by filling in the http response. Then it will pass the event to the configured output backend.
+In an [aggregated](aggregated-api-servers.md) deployment, the `kube-aggregator` is able to fill in
+`Metadata` level audit events, but not above. For the higher audit levels, an audit event is
+generated _both_ in the `kube-aggregator` and in the end-user apiserver. The events can be
+de-duplicated in the audit backend based on the audit ID, which is generated from the `X-Request-ID`
+header. The event generated by the end-user apiserver may not have the full authentication information.
```

Comment:
I added an `Audit Security` section to address this, as well as spoofing the source IP & tampering with the audit policy.